python/py-cachetools/Portfile

```diff
@@ -0,0 +1,28 @@
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-cachetools
+version             3.1.0
+checksums           rmd160  ba8042f518899a8a6fe0b1506741290cbf720812 \
+                    sha256  9efcc9fab3b49ab833475702b55edd5ae07af1af7a4c627678980b45e459c460 \
+                    size    20664
+
+license             MIT
+platforms           darwin
+supported_archs     noarch
+maintainers         nomaintainer
+
+description         memoizing collections and decorators
+long_description    This module provides various ${description}, including variants of the \
+                    Python 3 Standard Library @lru_cache function decorator.
+homepage            https://github.com/tkem/cachetools
+
+master_sites        pypi:[string index ${python.rootname} 0]/${python.rootname}
+distname            ${python.rootname}-${version}
+
+python.versions     27 34 35 36 37
```

Comment:
unless really necessary for another Port, we prefer not to add older Python 3 versions anymore. So if possibly use ```python.versions     27 37```